src/promptflow/promptflow/_sdk/entities/_connection.py

```diff
@@ -618,6 +632,41 @@ def _get_schema_cls(cls):
         return FormRecognizerConnectionSchema
 
 
+class CustomStrongTypeConnection(_Connection):
```

Comment:
Could you please help point out where is this class being used? didn't find the usage...

Reply:
![image](https://github.com/microsoft/promptflow/assets/46446115/ba65acde-88e7-4849-8bd1-de96dd57f817)
It is used when user write their own custom strong type connection class. This inheritance is in order to support SDK way of crud a connection, like this:
```
conn = MyFirstConnection(name=name, api_key="test", api_base="test")
_client.connections.create_or_update(conn)
```

Reply:
Got it.